src/app/preferences/views/APIKeys/APIKeyAdd/APIKeyAdd.test.tsx

```diff
@@ -1,10 +1,33 @@
-import { shallow } from "enzyme";
+import { screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { CompatRouter } from "react-router-dom-v5-compat";
+
+import { Label as APIKeyFormLabels } from "../APIKeyForm/APIKeyForm";
 
 import { APIKeyAdd } from "./APIKeyAdd";
 
+import type { RootState } from "app/store/root/types";
+import { rootState as rootStateFactory } from "testing/factories";
+import { renderWithMockStore } from "testing/utils";
+
 describe("APIKeyAdd", () => {
+  let state: RootState;
+
+  beforeEach(() => {
+    state = rootStateFactory();
+  });
+
   it("can render", () => {
-    const wrapper = shallow(<APIKeyAdd />);
-    expect(wrapper.find("APIKeyForm").exists()).toBe(true);
+    renderWithMockStore(
```

Comment:
You could use `renderWithBrowserRouter` in these tests to prevent having to set up the router.